Tasks/UseRubyVersionV0/task.json

```
@@ -4,7 +4,7 @@
     "friendlyName": "Use Ruby version",
     "description": "Use the specified version of Ruby from the tool cache, optionally adding it to the PATH",
     "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/use-ruby-version",
-    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=2005989)",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=2005989) or [see the Ruby documentation](https://www.ruby-lang.org/)",
```

We could link directly to the ruby documentation instead of the homepage
```suggestion
    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=2005989) or [see the Ruby documentation](https://www.ruby-lang.org/en/documentation/)",
```

---

Wherever possible, I tried not to link directly to English documentation. Nearly all tools had an option which was generic in the URL and redirected to English based on my browser. I couldn't find such a link for Ruby.